dispatch/modules/content/models.py

```diff
@@ -451,13 +459,76 @@ def save(self, **kwargs):
         super(Image, self).save(**kwargs)
 
         if is_new and self.img:
-            image = Img.open(StringIO.StringIO(self.img.read()))
+            s = self.img.read()
+            image = Img.open(StringIO.StringIO(s))
             self.width, self.height = image.size
 
-            super(Image, self).save()
+            # Exif data is only read from files with TIFF or JPEG format, otherwise an error will occur
+            if self.get_extension() in Image.EXIF_FORMATS:
+                exiftags = image._getexif()
+
+                if exiftags is not None:
+                    # exiftags.get() returns None if no entry exists for a given key or if the entry is empty
+
+                    # exiftags object is a dictionary indexed by numeric tags corresponding to particular fields we may be
+                    # interested in. For a full list: https://www.sno.phy.queensu.ca/~phil/exiftool/TagNames/EXIF.html
+                    self.caption = exiftags.get(270)
```

Comment:
The `Image` model doesn't have a caption field. I would just leave this out and stick with `title`. That should fix your tests as well. 

Captions are only set when images are used as attachments in articles/galleries since the caption might depend on where the image is being used.